collector/netclass_linux_rtnl.go

```diff
@@ -0,0 +1,387 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !nonetclass && linux
+// +build !nonetclass,linux
+
+package collector
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"net"
+	"regexp"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/jsimonetti/rtnetlink"
+	"github.com/mdlayher/netlink"
+	"github.com/mdlayher/netlink/nlenc"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs/sysfs"
+	"golang.org/x/sys/unix"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	netclassRTNLIgnoredDevices = kingpin.Flag("collector.netclass-rtnl.ignored-devices", "Regexp of net devices to ignore for netclass collector.").Default("^$").String()
```

Comment:
This flag name needs to match the collector name.

Also, the filename of this collector should match. `netclass_rtnl_linux.go`.

```suggestion
	netclassRTNLIgnoredDevices = kingpin.Flag("collector.netclass_rtnl.ignored-devices", "Regexp of net devices to ignore for netclass collector.").Default("^$").String()
```